classes/client.php

```diff
@@ -134,7 +134,7 @@ public static function get_records_without_subscription() {
     public static function get_record_by_domain(string $domain, string $clientid) {
         global $DB;
 
-        $clientidclause = ($clientid) ? "client.id = :clientid" : "1";
+        $clientidclause = ($clientid) ? "client.id = :clientid" : "TRUE";
```

Comment:
would it be cleaner to move the AND inside the ternary? so the right side is just an empty string?

Reply:
I had that to start with. Without the AND in the sql it looks a little strange.
Either way...